test/cmd_line/vsplit.test.ts

```diff
@@ -0,0 +1,58 @@
+import { ModeHandler } from '../../src/mode/modeHandler';
+import { setupWorkspace, cleanUpWorkspace, assertEqualLines, assertEqual } from './../testUtils';
+import { runCmdLine } from '../../src/cmd_line/main';
+import * as vscode from 'vscode';
+import { join } from 'path';
+import * as assert from 'assert';
+import { getAndUpdateModeHandler } from '../../extension';
+
+/**
+ * Waits for the number of text editors in the current window to equal the
+ * given expected number of text editors.
+ *
+ * @param numEditors Expected number of editors in the window
+ */
+async function WaitForEditors(numEditors: number): Promise<void> {
```

Comment:
Can you move this to `testUtils`? There's probably value in sharing it with other test classes.